collect_app/src/main/java/org/odk/collect/android/preferences/ExperimentalPreferencesFragment.java

```diff
@@ -6,13 +6,12 @@
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import androidx.fragment.app.FragmentActivity;
-import androidx.preference.PreferenceFragmentCompat;
 
 import org.odk.collect.android.R;
 import org.odk.collect.android.activities.CollectAbstractActivity;
 import org.odk.collect.android.injection.DaggerUtils;
 
-public class ExperimentalPreferencesFragment extends PreferenceFragmentCompat {
+public class ExperimentalPreferencesFragment extends BasePreferenceFragment {
```

Comment:
Ah nice. These were left over from when they were `androidx.` and everything else was just `android.`